tests/api/routers/test_filing_api.py

```diff
@@ -224,3 +224,108 @@ def test_verify_lei_dependency(self, mocker: MockerFixture):
         assert isinstance(http_exc.value, HTTPException)
         assert http_exc.value.status_code == 403
         assert http_exc.value.detail == "LEI 1234567890 is in an inactive state."
+
+    async def test_unauthed_get_contact_info(self, app_fixture: FastAPI, unauthed_user_mock: Mock):
+        client = TestClient(app_fixture)
+        res = client.get("/v1/filing/institutions/1234567890/filings/2024/contact-info")
+        assert res.status_code == 403
+
+    async def test_get_contact_info(self, mocker: MockerFixture, app_fixture: FastAPI, authed_user_mock: Mock):
+        mock = mocker.patch("entities.repos.submission_repo.get_contact_info")
+        mock.return_value = ContactInfoDAO(
+            id=1,
+            filing=1,
+            first_name="test_first_name_1",
+            last_name="test_last_name_1",
+            hq_address_street_1="address street 1",
+            hq_address_street_2="",
+            hq_address_city="Test City",
+            hq_address_state="TS",
+            hq_address_zip="12345",
+            phone="112-345-6789",
+            email="name_1@email.test",
+        )
+
+        client = TestClient(app_fixture)
+        res = client.get("/v1/filing/institutions/1234567890/filings/2024/contact-info")
+        result = res.json()
+
+        assert res.status_code == 200
+        assert result["id"] == 1
+        assert result["first_name"] == "test_first_name_1"
+        assert result["last_name"] == "test_last_name_1"
+        assert result["hq_address_street_1"] == "address street 1"
+        assert result["hq_address_street_2"] == ""
+        assert result["hq_address_city"] == "Test City"
+        assert result["hq_address_state"] == "TS"
+        assert result["hq_address_zip"] == "12345"
+        assert result["phone"] == "112-345-6789"
+        assert result["email"] == "name_1@email.test"
+
```

Comment:
Can we add a final test here like:
```
mock.return_value = None
res = client.get("/v1/filing/institutions/1234567890/filings/2024/contact-info")
assert res.status_code == 204
```
Just to verify if there's no contact info, the endpoint returns 204 NO_CONTENT?  Also, helps keep the code coverage up lol